x/mint/keeper/keeper_test.go

```diff
@@ -403,8 +403,9 @@ func (suite *KeeperTestSuite) TestDistributeToModule() {
 				actualMintModuleBalanceAmount := bankKeeper.GetBalance(ctx, accountKeeper.GetModuleAddress(types.ModuleName), tc.mintedCoin.Denom).Amount
 				actualRecepientModuleBalanceAmount := bankKeeper.GetBalance(ctx, accountKeeper.GetModuleAddress(tc.recepientModule), tc.mintedCoin.Denom).Amount
 
-				if tc.expectedError {
+				if tc.expectedError != nil {
 					suite.Require().Error(err)
+					suite.Require().Equal(tc.expectedError.Error(), err.Error())
```

Comment:
Any reason for not using `errors.As` here?

Reply:
With this test, the error is detail, so I think `Equal` is more suitable